### PR TITLE
cohort field - mv_downloads  LESQ-602

### PR DESCRIPTION
**Pull request by k-huggs**
## Description

Music year: 2018

- Updates GQL query and schemas
- Adds downloadsFixture2023
- updates isDownloadLegacy check

## Issue(s)

Fixes #LESQ-602

## How to test

1. Go to https://deploy-preview-2215--oak-web-application.netlify.thenational.academy

## Checklist

- [ ] Added or updated tests where appropriate
- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner
- [ ] Does this PR update a package with a breaking change

### CODE REVIEW

File: src/components/TeacherViews/LessonDownloads.view.tsx [hunk 1]
```diff
@@ -38,6 +39,7 @@ type LessonDownloadsProps =
         isLegacy: boolean;
```
Comment (JBR90) [inline src/components/TeacherViews/LessonDownloads.view.tsx:39]: Looks like this lesson data could be extracted to a type to reduce duplication

**Review by JBR90 — APPROVED** [review]
Looks good, one optional comment

**Review by jack-skerrett-oak — APPROVED** [review]
Old and new lesson downloads still seem to work.

**Pull request merged**